tests/shopify_python/test_shopify_styleguide.py

```diff
@@ -0,0 +1,34 @@
+import pylint.testutils
+
+from shopify_python import shopify_styleguide
+
+
+class TestShopifyStyleGuideChecker(pylint.testutils.CheckerTestCase):
+
+    CHECKER_CLASS = shopify_styleguide.ShopifyStyleGuideChecker
+
+    def test_disable_name_only(self):
+        tokens = pylint.testutils.tokenize_str("""
+        import os  # pylint: disable=unused-import
+        import os  # pylint: disable=unused-import,W0611
+        import os  # pylint: disable=W0611,C0302,C0303
+        import os  # pylint: disable=W0611
+        import os  #pylint:disable=W0611
+        import os  #  pylint:\t\t  disable  \t\t    = \t\t W0611
+        def fnc():
+            # pylint: disable=C0112
+            pass
+        """.strip())
+
+        with self.assertAddsMessages(*[
+            pylint.testutils.Message('disable-name-only', line=line, args={'code': code, 'name': 'unknown'})
```

Comment:
In tests it's not a real linter, it's a [`pylint.testutils.UnittestLinter`](https://github.com/PyCQA/pylint/blob/3ccad8c270f2b74325dab2a9ff4eac51edfc41c0/pylint/testutils.py#L159).